jobbergate-api/jobbergate_api/apps/services.py

```diff
@@ -585,15 +575,21 @@ async def _get_file_data_from_url(self, file_url: AnyUrl) -> io.BytesIO:
                     raise_exc_class=ServiceError,
                     raise_kwargs=dict(status_code=status.HTTP_400_BAD_REQUEST),
                 )
-
-                session = Session()
-                async with session.resource("s3", url_string) as s3:
-                    bucket = await s3.Bucket(bucket_name)
-                    s3_obj = await bucket.Object(key)
-                    await s3_obj.download_fileobj(file_obj)
+                file_url_string = f"https://{bucket_name}.amazonaws.com{key}"
```

Comment:
Creating a separated discussion... check my comment [here](https://github.com/omnivector-solutions/jobbergate/pull/640/files#r1817085106) please

Reply:
Jeez, I totally missed it. 🤦🏼

Reply:
Addressed in commit a8e5887